src/test/add-remove-todo.test.js

```diff
@@ -0,0 +1,17 @@
+import Todo from '../TodoClass.js';
+
+describe('Test add functionality', () => {
+  test('Add 1 todo Item', () => {
+    const todo = new Todo('Make tests for this!!');
+    expect(Todo.getAllTodos().length).toBe(1);
+    todo.deleteTodo();
+  });
+});
+
+describe('Test remove functionality', () => {
+  test('Remove todo task selected by the user', () => {
+    const removeTask = new Todo('ensure project test work without errors!');
+    removeTask.deleteTodo();
+    expect(removeTask.getAllTodos().length).toBe(0);
+  });
```

Comment:
- [ ] your remove test is not passing , kindly fix the issue 🙏 


![image](https://user-images.githubusercontent.com/89282221/207603160-360a3ae4-d2a0-41d5-9c3e-6efe48da23ea.png)